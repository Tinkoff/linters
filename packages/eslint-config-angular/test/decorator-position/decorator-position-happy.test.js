import ESlint from 'eslint';
import path from 'path';

describe('decorator-position / happy path', () => {
  const cli = new ESlint.CLIEngine({
    cwd: path.join(__dirname, '..'),
    useEslintrc: false,
    baseConfig: {
      extends: ['../decorator-position'],
    },
  });

  it('happy', () => {
    const codeframe = cli.getFormatter('codeframe');

    const report = cli.executeOnFiles([
      path.join(
        __dirname,
        './__fixtures__/decorator-position-happy.fixture.ts'
      ),
    ]);

    expect(codeframe(report.results)).toMatchSnapshot();
  });
});
