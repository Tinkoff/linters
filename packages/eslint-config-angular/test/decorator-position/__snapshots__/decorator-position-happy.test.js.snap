// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`decorator-position / happy path happy 1`] = `""`;
